project/src/components/app/app.test.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import {Router} from 'react-router-dom';
+import {createMemoryHistory} from 'history';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import { ALL_GENRES, AuthorizationStatus, AppRoute} from '../../const';
+import App from './app';
+
+let history = null;
+let store = null;
+let storeForNoAuth = null;
+let fakeApp = null;
+let fakeAppNoAuth = null;
+
+const film = {
+  id: '1',
+  imgName: 'https://some-name.jpg',
+  name: 'The Grand Budapest Hotel',
+  posterImage: 'img/the-grand-budapest-hotel-poster.jpg',
+  previewImage: 'img/the-grand-budapest-hotel.jpg',
+  backgroundImage: 'img/the-grand-budapest-hotel-bg.jpg',
+  backgroundColor: '#ffffff',
+  videoLink: 'https://some-link',
+  previewVideoLink: 'https://some-link',
+  description: 'In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H.',
+  rating: 8.9,
+  scoresCount: 240,
+  director: 'Wes Andreson',
+  starring: ['Bill Murray', 'Edward Norton', 'Jude Law', 'Willem Dafoe', 'Saoirse Ronan'],
+  runTime: 99,
+  genre: 'Comedy',
+  released: 2014,
+  isFavorite: false,
+};
+
+const films = [
+  {
```

Comment:
можно было просто написать:
```suggestion
const films = [film]
```